german/Smartphone/smartphone.md

```diff
@@ -1,11 +1,13 @@
 Gründlich durchsuchst du all deine Hosentaschen...
-in der zuletzt geprüften Tasche findest du es. Hastig schaltest du das Smartphone 
+in der zuletzt geprüften Tasche findest du es. Hastig schaltest du das Smartphone
 ein um zu prüfen wie voll der Akku noch ist und ob du Internetempfang hast...
 
 Nachdem das Smartphone gebootet wurde siehst du...
 
+[dass du seit einem Monat nicht mehr trainiert hast!](Gym/gym.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.